pussy_snapshot_deploy_guide.md

```diff
@@ -0,0 +1,83 @@
+# ![](https://ipfs.io/ipfs/QmWeAd87fZm1pMYyX9BmhnTrXYKCZLoyzMJMcSwNHfB6gU) Deploy Pussy node from snapshot
+
+## Download snapshot
+First, go to [Pussy Snapshot Robot](https://cyb.ai/network/bostrom/contract/pussy137p3ll78yrlw3gtfltgwhdkz0qxke4z638md6n) quitter page
```

Comment:
Buy default on https://cyb.ai opens Bostrom network. It should be switched to pussy manualy as far as I can understand.

Reply:
Thanks! I think it might be a front problem
https://github.com/cybercongress/cyb/issues/844